src/Core/Services/Implementations/UserService.cs

```diff
@@ -690,7 +690,7 @@ public async Task<IdentityResult> AdminResetPasswordAsync(OrganizationUserType c
 
             user.RevisionDate = user.AccountRevisionDate = DateTime.UtcNow;
             user.Key = key;
-            user.ForcePasswordReset = true;
+            /* user.ForcePasswordReset = true; */
```

Comment:
Let's add a comment in the code here as well @addisonbeck , thanks for the suggestion Daniel!